datahub-web-react/src/app/entity/mlModel/profile/MLModelSummary.tsx

```diff
@@ -26,9 +60,63 @@ export default function MLModelSummary() {
         },
     ];
 
+    const formatDate = (timestamp?: number) => {
+        if (!timestamp) return '-';
+        const milliseconds = timestamp < 10000000000 ? timestamp * 1000 : timestamp;
+        return new Date(milliseconds).toISOString().slice(0, 19).replace('T', ' ');
+    };
+
+    const renderTrainingJobs = () => {
+        const lineageTrainingJobs = model?.properties?.mlModelLineageInfo?.trainingJobs || [];
+        
+        if (lineageTrainingJobs.length === 0) return '-';
+        
+        return lineageTrainingJobs.map((jobUrn, index) => (
+            <div key={jobUrn}>
+                <JobLink to={entityRegistry.getEntityUrl(EntityType.DataProcessInstance, jobUrn)}>
+                    {jobUrn}
+                </JobLink>
+                {index < lineageTrainingJobs.length - 1 && ', '}
+            </div>
+        ));
+    };
+
     return (
         <TabContent>
             <Space direction="vertical" style={{ width: '100%' }} size="large">
+            <Typography.Title level={3}>Model Details</Typography.Title>
+                <InfoItemContainer justifyContent="left">
+                    {/* TODO: should use versionProperties? */}
+                    <InfoItem title="Version">
+                        <InfoItemContent>{model?.properties?.version}</InfoItemContent>
+                    </InfoItem>
+                    <InfoItem title="Registered At">
+                        <InfoItemContent>{formatDate(model?.properties?.created?.time)}</InfoItemContent>
+                    </InfoItem>
+                    <InfoItem title="Last Modified At">
+                        <InfoItemContent>{formatDate(model?.properties?.lastModified?.time)}</InfoItemContent>
+                    </InfoItem>
+                </InfoItemContainer>
+                <InfoItemContainer justifyContent="left">
+                    <InfoItem title="Created By">
+                        <InfoItemContent>{model?.properties?.created?.actor}</InfoItemContent>
+                    </InfoItem>
+                    <InfoItem title="Aliases">
+                        <InfoItemContent>
+                            {/* use versionProperties for aliases */}
```

Comment:
commenting this out until this get merged : https://github.com/datahub-project/datahub/pull/12166